packages/tooltip/src/Tooltip/Tooltip.tsx

```diff
@@ -88,7 +90,7 @@ function Tooltip({
   align = 'top',
   justify = 'start',
   spacing = 12,
-  usePortal = true,
+  renderMode = RenderMode.TopLayer,
```

Comment:
When using `topLayer` or `inline`, the tooltip inherits the styles of the parents. For example, the tooltip is in uppercase in the story file because the button is uppercase, but it probably should not be uppercase.

Reply:
I wonder if this is more of a new consumer responsibility since the tooltip is now a child of the trigger.

Reply:
I think we should ask design because i'm not sure.